src/nimble.nim

```diff
@@ -456,11 +456,13 @@ proc installFromDir(dir: string, requestedVer: VersionRange, options: Options,
   # processDeps).
   saveNimbleData(options)
 
+  # update package path to point to installed directory rather than the temp directory
+  pkgInfo.myPath = dest
+
   # Return the dependencies of this package (mainly for paths).
   result.deps.add pkgInfo
   result.pkg = pkgInfo
   result.pkg.isInstalled = true
```

Comment:
on closer inspection this line, `.isInstalled = true`, probably should also be done before the `pkgInfo` var is added to `result.deps` and set to `result.pkg`. though this isn't necessary for fixing the issue at hand, it is probably the correct thing to do. i can add another commit to this PR that makes this change too if you think that is appropriate, @dom96.